src/main/java/com/pinterest/secor/main/LogFileVerifierMain.java

```diff
@@ -49,11 +49,13 @@ private static CommandLine parseArgs(String[] args) throws ParseException {
                .create("t"));
         options.addOption(OptionBuilder.withLongOpt("start_offset")
                .withDescription("offset identifying the first set of files to check")
+               .hasArg()
```

Comment:
it was just returning null and failing to parse before. So basically NullPointerException. 
It doesn't seem to be making it a required argument because I have jobs that are running without the start and end arguments.